src/translations/sqlb_pt_BR.ts

```diff
@@ -4306,137 +4354,139 @@ Warning: not all data has been fetched from the table yet due to the partial fet
     </message>
     <message>
         <source>&amp;Quotes for identifiers</source>
-        <translation type="unfinished"></translation>
+        <translation>Áspas &amp;para identificadores</translation>
     </message>
     <message>
         <source>Choose the quoting mechanism used by the application for identifiers in SQL code.</source>
-        <translation type="unfinished"></translation>
+        <translation>Escolha as áspas utilizadas pela aplicação para identificadores no código SQL.</translation>
     </message>
     <message>
         <source>&quot;Double quotes&quot; - Standard SQL (recommended)</source>
-        <translation type="unfinished"></translation>
+        <translation>&quot;Áspas duplas&quot; - SQL Padrão (recomendado)</translation>
     </message>
     <message>
         <source>`Grave accents` - Traditional MySQL quotes</source>
-        <translation type="unfinished"></translation>
+        <translation>`Crases`- MySQL tradicional</translation>
     </message>
     <message>
         <source>[Square brackets] - Traditional MS SQL Server quotes</source>
-        <translation type="unfinished"></translation>
+        <translation>[Colchetes] - MS SQL Server tradicional</translation>
     </message>
     <message>
         <source>Keywords in &amp;UPPER CASE</source>
-        <translation type="unfinished"></translation>
+        <translation>Palavras-chave em &amp;CAIXA ALTA</translation>
     </message>
     <message>
         <source>When set, the SQL keywords are completed in UPPER CASE letters.</source>
-        <translation type="unfinished"></translation>
+        <translation>Quando definido, as palavras-chave SQL serão completadas em CAIXA ALTA.</translation>
     </message>
     <message>
         <source>When set, the SQL code lines that caused errors during the last execution are highlighted and the results frame indicates the error in the background</source>
-        <translation type="unfinished"></translation>
+        <translation>Quando definido, as linhas de código SQL que causaram erros durante a última execução são destacadas e os resultados indicam os erros no fundo</translation>
     </message>
     <message>
         <source>Are you sure you want to clear all the saved settings?
 All your preferences will be lost and default values will be used.</source>
-        <translation type="unfinished"></translation>
+        <translation>Você tem certeza que deseja limpar as configurações salvas?
+Todas as suas preferências serão perdidas e os valores padrão serão utilizados.</translation>
     </message>
     <message>
         <source>When enabled, the line breaks in the Schema column of the DB Structure tab, dock and printed output are removed.</source>
-        <translation type="unfinished"></translation>
+        <translation>Quando ativado, as quebras de linha na coluna Schema da aba Estruturas do BD e nas saídas impressas são removidas.</translation>
```

Comment:
A pair of consistency problems: The column is translated in "Estruturas do bando de dados" as "Esquema". Here "Estruturas do BD" and "Schema".

![imagen](https://user-images.githubusercontent.com/13812910/46976524-01627100-d0ca-11e8-8ea4-6580ecc727a9.png)